tools/individual-pkg-gen/transform-packages.ts

```diff
@@ -100,6 +100,10 @@ function transformPackageJson(pkg: any, source: string, destination: string, alp
   packageJson.name += '-alpha';
   packageJson.repository.directory = `packages/individual-packages/${pkgUnscopedName}`;
 
+  // All individual packages are private by default on v2.
+  // This needs to be removed for the alpha modules to be published.
+  packageJson.private = undefined;
```

Comment:
On `master`, the `private` field is not set on any of the original (`@aws-cdk/aws-foobar`) service `package.json`s, because we _are_ publishing those packages individually. On `v2-main`, however, we're currently only publishing a limited set of packages, so all of the original versions of the alpha modules have `private` set. This needs to be un-set in order for these packages to be public.

However, this caused me to look closer at how we determine what packages to pack. We use `lerna ls --toposort --json` to list the packages to pack.

https://github.com/aws/aws-cdk/blob/d21561adbd39543516cdf69026849877a0348222/scripts/list-packages#L15

This doesn't appear (currently) to pick up any of the alpha modules. Un-setting `private` on the `individual-packages` package, and adding that path to `lerna.json` doesn't appear to fix it either. I'll keep digging!

Reply:
Ah, the `individual-packages` directory was missing from our `workspaces` list and in `lerna.json`. Added it to both, which is now showing the results I'd expect.